tests/sched/sched_test.go

```diff
@@ -0,0 +1,91 @@
+package tests
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/portworx/torpedo/drivers/node"
+	"github.com/portworx/torpedo/drivers/scheduler"
+	. "github.com/portworx/torpedo/tests"
+)
+
+func TestStopStartScheduler(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Torpedo: StopStartScheduler")
+}
+
+var _ = BeforeSuite(func() {
+	InitInstance()
+})
+
+var _ = Describe("{StopScheduler}", func() {
+	testName := "stopscheduler"
+	It("has to stop scheduler service and check if all applications are fine", func() {
+		var err error
+		var contexts []*scheduler.Context
+		for i := 0; i < Inst().ScaleFactor; i++ {
+			contexts = append(contexts, ScheduleAndValidate(fmt.Sprintf("%s-%d", testName, i))...)
+		}
+
+		Step("get nodes for all apps in test and induce drive failure on one of the nodes", func() {
+			for _, ctx := range contexts {
+				var appNodes []node.Node
+
+				Step(fmt.Sprintf("get nodes where %s app is running", ctx.App.Key), func() {
+					appNodes, err = Inst().S.GetNodesForApp(ctx)
+					Expect(err).NotTo(HaveOccurred())
+					Expect(appNodes).NotTo(BeEmpty())
+				})
+
+				Step(fmt.Sprintf("stop scheduler service"), func() {
+					for _, nodeToStopService := range appNodes {
+						err := Inst().S.StopSchedOnNode(nodeToStopService)
+						Expect(err).NotTo(HaveOccurred())
+					}
+					Step("wait for the service to stop", func() {
```

Comment:
Did this test pass on your run?

For a repl 3 application (e.g nginx), `appNodes` will be all the nodes in the cluster. If you stop the scheduler on all nodes, the app has nowhere to go.

The test needs to 

1. pick only one node for an app
2. Stop scheduler on that node
3. Wait and validate the app is back healthy (time for eviction threshold to cross and new pod to come up)
4. Start scheduler on that node
5. Move to the next app ctx